en/frames_plane/reptile_dragon_2.md

```diff
@@ -0,0 +1,344 @@
+# Reptile Dragon 2 (RD2) Build
+
+![Finished Reptile Dragon 2 airframe front](../../assets/airframes/fw/reptile_dragon_2/airframe_front.jpg)
+
+![Finished Reptile Dragon 2 airframe rear](../../assets/airframes/fw/reptile_dragon_2/airframe_rear.jpg)
+
+The Reptile Dragon 2 is a twin motor RC airplane specifically designed for efficient FPV [(first person view)](https://en.wikipedia.org/wiki/First-person_view_(radio_control)) flying.
+Being specific for FPV, the RD2 is optimized for easy mounting of cameras, sensors, logic electronics, large batteries, antennas, and other payload components which would be found on a typical FPV airplane. 
+This emphasis on payload makes this airplane an ideal candidate for a PX4 installation.
+
+## Overview
+
+The goal of this build was to create an efficient, long endurance FPV platform to be used for general PX4 testing and development.
+
+Key airframe features:
+
+- Spacious interior
+- Easy access to the entire fuselage cavity with large top hatch
+- Rear hatch
+- Removable V tail or conventional tail options included
+- Threaded inserts in the wings and fuselage top for external mounting
+- Numerous mounting features
+  - Top antenna hole
+  - Top GPS cover
+  - Side "T" antenna mounts
+  - Rear electronics tray
+  - Front facing "action cam" cutout
+  - Front facing FPV camera cutout 
+- Removable wings
+- Low stall speed
+- Gentle handling 
+
+Key build features
+
+- Easy overall build
+- Easy access to Pixhawk and all peripherals
+- FPV with camera pan mount
+- Air data from pitot/static probe
+- ~40 minute long flight times
+
+## Parts list
+
+- [Reptile Dragon 2 kit](https://usa.banggood.com/REPTILE-DRAGON-2-1200mm-Wingspan-Twin-Motor-Double-Tail-EPP-FPV-RC-Airplane-KIT-or-PNP-p-1805237.html?cur_warehouse=CN&ID=531466)
+- [Pixhawk 5x Carrier board](https://shop.holybro.com/pixhawk-5x_p1279.html)
+- [ARK6X FMU](https://arkelectron.com/product/arkv6x/)

```

Comment:
@chris1seto Just to be clear, you are using the carrier board for the Pixhawk 5x from Holybro with the AR6X FMU - what's the benefit of that? I mean wouldn't it be easier just to go "all holybro" here? Or was there a cost issue or something else.

It's fine, just first question that came into my head.

Reply:
Basically what happened was the airplane was originally set up for a 5x carrier and Fmu, but Alex had just finished and sent me one of his first 6x's to test. He hadn't finished the carrier board yet, so the majority of the airplane's life up until now had that combination. He just sent me the finished carrier board, so I plan to edit the build log to reflect that being installed now. So basically, the build log reflects his progress on developing the 6x and carrier board and finishing them at different times,